src/esp/metadata/attributes/PhysicsManagerAttributes.h

```diff
@@ -64,15 +62,15 @@ class PhysicsManagerAttributes : public AbstractAttributes {
   std::string getObjectInfoInternal() const override {
     return getSimulator()
         .append(1, ',')
-        .append(cfg.value("timestep"))
+        .append(std::to_string(getTimestep()))
         .append(1, ',')
-        .append(cfg.value("max_substeps"))
+        .append(std::to_string(getMaxSubsteps()))
         .append(1, ',')
-        .append(cfg.value("gravity"))
+        .append(getAsString("gravity"))
         .append(1, ',')
-        .append(cfg.value("friction_coefficient"))
+        .append(std::to_string(getFrictionCoefficient()))
         .append(1, ',')
-        .append(cfg.value("restitution_coefficient"));
+        .append(std::to_string(getRestitutionCoefficient()));
   }
```

Comment:
Here as well :P

Reply:
I'm going to strip out all of the offending functionality from this PR and refactor all of these to use the formatString process in another PR.